app/src/main/java/org/kiwix/kiwixmobile/main/KiwixMainActivity.kt

```diff
@@ -66,6 +99,13 @@ class KiwixMainActivity : CoreMainActivity() {
   }
 
   override fun onBackPressed() {
+    if (new_navigation_container.isDrawerOpen(drawer_nav_view)) {
```

Comment:
stuff like this isn't in `Core`? Does the drawer close when you press back in Custom?